templates_ark/example_pixel_clustering.ipynb

```diff
@@ -174,7 +174,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Rather than having this as None, we should tell the user to skip the cell if they don't want to run it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/653/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>